dtests/src/test/java/io/snappydata/hydra/misc/snap3028.conf

```diff
@@ -0,0 +1,34 @@
+INCLUDE $JTESTS/io/snappydata/hydra/northwind/startEmbeddedModeCluster.conf;
+
+INITTASK     taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSQLScripts
+    io.snappydata.hydra.cluster.SnappyPrms-sqlScriptNames = "create_and_load_table_snap3028.sql"
+    threadGroups = leadThreads;
+
+INITTASK    taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSnappyJob
```

Comment:
No need to add two INITTASKS. Instead add a task with maxTimesToRun=2 and use isSerialExecution=true to run the tasks sequentially.